src/stream/src/executor/source/source_executor.rs

```diff
@@ -752,12 +752,14 @@ struct WaitCheckpointTaskBuilder {
 
 impl WaitCheckpointTaskBuilder {
     fn update_task_on_chunk(&mut self, offset_col: ArrayRef) {
-        #[expect(clippy::single_match)]
         match &mut self.building_task {
             WaitCheckpointTask::AckPubsubMessage(_, arrays) => {
                 arrays.push(offset_col);
             }
-            _ => {}
+            WaitCheckpointTask::AckNatsJetStream(_, arrays, _) => {
+                arrays.push(offset_col);
+            }
```

Comment:
We does this `offset_col` mean?

Reply:
Oh, good catch, `offset_col` is still the seq id and reply_subject is in MessageMeta.